test/invariants/fuzz/Protocol.t.sol

```diff
@@ -219,7 +220,10 @@ contract FuzzProtocol is EchidnaTest {
 
       // 19
       assert(pool.isFinalized());
-    } catch {
+    } catch (bytes memory errorData) {
+      if (keccak256(errorData) == IBPool.BPool_SpotPriceAfterBelowSpotPriceBefore.selector) {
+        assert(false);
```

Comment:
we could add a comment saying that we're trying to hit this error but seems unreachable, or is it within `19`?